src/index.html

```diff
@@ -7,7 +7,53 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="container">
+    <div class="stars--1">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars--2">
+      <div class="stars__star star__filled"></div>
```

Comment:
You should not add **star_filled** class, try to style the star image based on the class from parent with a modifier for example **stars--4** and then you can style nth-child in css. Remember to group selectors with comma - point 2 - https://github.com/mate-academy/layout_stars/blob/master/checklist.md